examples/make_simple/BUILD.bazel

```diff
@@ -3,10 +3,9 @@ load("@rules_foreign_cc//foreign_cc:defs.bzl", "make")
 
 make(
     name = "make_lib",
-    build_data = ["//make_simple/code:clang_wrapper.sh"],
-    env = {
-        "CLANG_WRAPPER": "$(execpath //make_simple/code:clang_wrapper.sh)",
-    },
```

Comment:
IIRC the clang_wrapper.sh script is present here to test the functionality of referencing a build tools from the environment so I'd rather this wasn't removed.

Reply:
I reintroduced the wrapper and along the way fixed the test failure on Windows.